src/jit.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * rv32emu is freely redistributable under the MIT License. See the file
+ * "LICENSE" for information on usage and redistribution of this file.
+ */
+
+#pragma once
+
+#include <assert.h>
```

Comment:
Remove the inclusion of `<assert.h>`, `<stdio.h>`, `<string.h>`, and `<unistd.h>` because they are not really used in this header.